src/components/Strings.js

```diff
@@ -1,19 +1,19 @@
 export function NSamples({ total }) {
   if (!total) {
     return 'No Samples';
-  } else if (total === 1) {
+  }
+  if (total === 1) {
```

Comment:
Could `total` be 0?

Reply:
yes, and in that case this should return `'No Samples'`

Reply:
Thanks. I didn't see the first `if` block.